### ISSUE TITLE
能否增加dnscrypt-proxy可用的格式

### ISSUE THREAD

**Issue by CNMan** [googlehosts/hosts#170]
格式参考:https://github.com/jedisct1/dnscrypt-proxy/raw/master/dnscrypt-proxy/example-cloaking-rules.txt
https://github.com/googlehosts/hosts/raw/master/hosts-files/hosts
域名和IP两列互换即可

**Comment by beyondgfw** [comment]
已加，麻烦测试下

**Comment by CNMan** [comment]
可用，谢谢
已添加到 https://github.com/CNMan/dnscrypt-proxy-config

**Comment by sergeevabc** [comment]
Could you be so kind to tell (in plain English) what is the purpose of this ```hosts``` file?
Looks like you want to prefer some IPs for domains instead of ones that a DNS server returns, right?

**Comment by SerCom-KC** [comment]
> instead of ones that a DNS server returns

[DNS poisoning](https://en.m.wikipedia.org/wiki/DNS_spoofing) is very common in China, simple as that.
We're too lazy to explain this in detail so please serve yourself with [some Wikipedia article](https://en.wikipedia.org/wiki/Great_Firewall#Current_methods).
